src/main/java/uk/ac/cam/cl/dtg/isaac/quiz/IQuestionAttemptManager.java

```diff
@@ -3,7 +3,7 @@
 import uk.ac.cam.cl.dtg.isaac.dos.LightweightQuestionValidationResponse;
 import uk.ac.cam.cl.dtg.isaac.dos.QuestionValidationResponse;
 import uk.ac.cam.cl.dtg.isaac.dos.users.Role;
-import uk.ac.cam.cl.dtg.segue.api.Constants.TimeInterval;
```

Comment:
Minor point, but is there any specific reason this has been changed? TimeInterval is still being imported directly in other files.

Reply:
Looks like this happened with an import that removed it. Will change it back.